### ISSUE TITLE
report assetic bug

### ISSUE THREAD

**Issue by nimasdj** [symfony/AsseticBundle#340]
@stof 
when I go to 
http://www.mysite.com/symfony/web/app.php/blah
and also to 
http://www.mysite.com/symfony/web/app_dev.php/blah
everything works fine but if I move app_dev.php to
http://www.mysite.com/symfony/index.php/blah
(I renamed it to index.php too)
and also applying the correct include path for /app/bootstrap.php.cache and /app/autoload.php in this index.php, everything works fine and css is loaded too but css background image is pointed wrongly to:
http://www.mysite.com/bundles/applicationadmin/images/top_bg.jpg 404 (Not Found)
instead of 
http://www.mysite.com/symfony/web/bundles/applicationadmin/images/top_bg.jpg
Have you ever tested/tried this?

**Comment by nimasdj** [comment]
please read "UPDATE" section above.

**Comment by nimasdj** [comment]
@stof please read "UPDATE" section above.

**Comment by stof** [comment]
@nimasdj if your web root is the root of the project rather than the web folder (which might create issues btw), you should read from the root too. Otherwise, Assetic cannot guess that you have a mismatch between its config and your actual document root.

**Comment by nimasdj** [comment]
@stof 
Do you mean I have to have /symphony/web/bundles, /symphony/web/images, /symphony/web/css directly in /symphony/ folder? I actually tried that too, but if I add:
```yaml
write_to: "%kernel.root_dir%/../../public_html"
```
or 
```yaml
write_to: "%kernel.root_dir%/.."
```
then I run assets:install, it still generates 'bundles' in /symphony/web rather than in /symphony/.
Please advice.

**Comment by stof** [comment]
No, what I mean is that you should have ``read_from`` set to ``"%kernel.root_dir%/.."`` and include the ``web/`` part in your asset paths when referencing them if your document root is not the web folder but its parent

**Comment by nimasdj** [comment]
@stof I added:
```yaml
 read_from:  "%kernel.root_dir%/.."
```
and also changed the assets too:
```twig
{% stylesheets 'web/bundles/applicationadmin/css/*' filter='cssrewrite' %}<link rel="stylesheet" href="{{ asset_url }}" type="text/css" />{% endstylesheets %}
```
then deleting cache to run assetic:dump /[root]/images/ and [root]/css are generated. css and images are loaded now. but the problem is that css background image is pointed to:
http://www.mysite.com/web/bundles/applicationadmin/css/images/top_bg.jpg 404 (Not Found)
rather than
http://www.mysite.com/symfony/web/bundles/applicationadmin/css/images/top_bg.jpg 404 (Not Found)
path in /web/bundles/appbundles/css/style.css is relative and is '../images/blah.jpg'
1) what else to do solve this css background image?
2) how to keep /images/ and /css/ being dump in /web/ folder yet rather than root?
Please help.

**Comment by nimasdj** [comment]
@stof please read update section above.

**Comment by nimasdj** [comment]
@stof I think there is another issue (probably a bug?) too:
with doing this all, the only problem is css background images as I said above, but the doctrine query runs as expected and I see query result on page. but the profiler/debugger shows the number of queries as 0. why? why moving app_dev.php from /web/ folder to parent folder should affect profiler too? (I deleted the cache.)

**Comment by nimasdj** [comment]
@stof Any advice yet? If this is not confirmed as a bug, please advice. I am stuck.

**Comment by trueplotnick** [comment]
Have the same issue.
I created test bundle - "StyleTestBundle", than added to Resources\public\css file test.css.

.test {
    height: 400px;
    border: 1px solid black;
    background-image: url(img/test.png);
}

in css folder i created img folder and added file test.png.

resources
      --public
             ---css
                    --img
                         --test.png
                    --test.css

in template:

{% stylesheets
                'bundles/styletest/css/*' filter='cssrewrite' %}
          <link rel="stylesheet" type="text/css" media="screen" href="{{ asset_url }}" />
{% endstylesheets %}

After that i use assetic:install. My project root named symfony. In folder symfony/web i get subfolder

bundles/styletest/css/ with test.css file and 
bundles/styletest/css/img with test.ong file.

When i run <hostname>/symfony/app_dev.php it works correctly. But if i use prod link - <hostname>/symfony - the background image (test.png) is not display.

Both image links in css files for prod and dev are the same.

../../bundles/styletest/css/img/test.png

But referers are different.

http://localhost/symfony/css/e9c43d1.css - for prod

http://localhost/symfony/app_dev.php/css/e9c43d1_1.css - for dev

And final links for image are different for prod and dev.

http://localhost/bundles/kendoui/css/img/test.png - for prod ( 404 not found)
http://localhost/symfony/bundles/kendoui/css/img/test.png - for dev (it works)
